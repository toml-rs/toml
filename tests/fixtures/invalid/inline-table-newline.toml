[package]
name = "foo"
version = "0.0.0"
authors = []

[target.x86_64-apple-darwin.freetype]
native = {
  foo: "bar"
}
