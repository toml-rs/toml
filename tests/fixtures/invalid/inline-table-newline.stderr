TOML parse error at line 7, column 11
  |
7 | native = {
  |           ^
Unexpected `
`
Expected key
